CMakeLists.txt

```
@@ -508,6 +508,9 @@ find_package(yaml-cpp 0.6.2 REQUIRED HINTS ${YAML_CPP_ROOT})
 get_filename_component(yaml_cpp_dir "${YAML_CPP_INCLUDE_DIR}" ABSOLUTE)
 get_filename_component(yaml_cpp_dir "${yaml_cpp_dir}" DIRECTORY)
 
+
+find_package(spdlog 1.8.1 REQUIRED)
```

```suggestion
find_package(spdlog 1.5 REQUIRED)
```

On Debian Buster, the newest available backport for `spdlog` is 1.5.0: https://packages.debian.org/buster-backports/libspdlog-dev 

Supporting Debian Buster is a necessary requirement for our deployments at the moment.

---

resolved in the latest changes